src/Counter.js

```diff
@@ -0,0 +1,16 @@
+import React, {useState} from "react";
+
+const Counter = () => {
+
+    const [counter, setCounter] = useState(0);
+
+
```

Comment:
this is relevant?

Reply:
> this is relevant?

My mistake :)
Its the counter state that Daniel showed us while we did the live coding and forgot to delete it.